PHPCSStandards/PHPCSUtils
# Release 1.0.0-alpha4

## Release checklist

### Update website

- [x] Regenerate the PHPDoc documentation - PR #396
    :pencil2: Clear out the `docs/phpdoc` directory and then use phpDocumentor 3 with the command `phpdoc`
- [x] Sync any changes in the Readme into the website `index.md` file. - PR #396
    :pencil2: Copy & paste the content from the `README.md` file to `docs/index.md` (and double-check the few remaining differences are intact).
    To verify the output locally:
    ```bash
    bundle update
    bundle exec jekyll serve
    ```
    and then visiting <http://localhost:4000/> to see the result.

### General

- [x] Verify, and if necessary, update the version constraints for dependencies in the `composer.json` - N/A
- [ ] Verify that any new functions have type declarations whenever possible.
- [x] Add changelog for the release - PR #398
    :pencil2: Remember to add a release link at the bottom and to adjust the link for "Unreleased"!

### Release

- [x] Make sure all CI builds are green.
- [x] Verify that the website regenerated correctly.
- [x] Tag the release (careful, GH defaults to `develop`!).
- [x] Create a release from the tag (careful, GH defaults to `develop`!) & copy & paste the changelog to it.
    :pencil2: Don't forget to copy the link collection from the bottom of the changelog!
- [x] Close the milestone
- [x] Open a new milestone for the next release
- [x] If any open PRs/issues which were milestoned for this release did not make it into the release, update their milestone.
- [x] Fast-forward `develop` to be equal to `stable`

### Publicize

- [ ] Tweet about the release.
- [x] Inform the primary dependants of this repo (PHPCSExtra, WordPressCS, PHPCompatibility and VariableAnalysis) about the release.
